### PR TITLE
[KOGITO-5965] Fix migration to Trusty PMML 7.60

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @Ginxo

Needed by

https://github.com/kiegroup/kogito-runtimes/pull/1628

### CODE REVIEW

File: kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/model/KiePMMLOutputField.java [hunk 1]
```diff
@@ -56,7 +56,8 @@ public static Builder builder(String name, List<KiePMMLExtension> extensions) {
     static Optional<Object> getValueFromKiePMMLNameValuesByVariableName(final String variableName,
                                                                         final List<KiePMMLNameValue> kiePMMLNameValues) {
         return kiePMMLNameValues.stream()
-                .filter(kiePMMLNameValue -> kiePMMLNameValue.getName().equals(variableName))
+                .filter(kiePMMLNameValue -> kiePMMLNameValue.getName().equals(variableName) &&
```
Comment (Ginxo) [inline kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/model/KiePMMLOutputField.java]:
```suggestion
                .filter(kiePMMLNameValue -> kiePMMLNameValue.getValue() != null &&
```

File: kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/model/KiePMMLOutputField.java [hunk 2]
```diff
@@ -56,7 +56,8 @@ public static Builder builder(String name, List<KiePMMLExtension> extensions) {
...
+                        kiePMMLNameValue.getValue() != null)
```
Comment (Ginxo) [inline kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/model/KiePMMLOutputField.java]:
```suggestion
                        kiePMMLNameValue.getName().equals(variableName))
```

**Review by Ginxo — APPROVED** [review]
LGTM, I just propose to switch checks (in order to avoid possible NullPointerException for `kiePMMLNameValue.getName()` in case `kiePMMLNameValue.getValue()` is also null)

File: kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/model/KiePMMLOutputField.java (hunk 1 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/model/KiePMMLOutputField.java]: @Ginxo 
Done, thx!

**Pull request closed without merging**